### PR TITLE
Introducing Cognitive Services

### PR DESCRIPTION
**Pull request by daniel-ms**
Adding Cognitive Services into Azure SDK.

This PR comprises of the new module - CognitiveServices - used to manage management of Cognitive Services accounts (Face API, Recommendations, etc.).

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@daniel-ms__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (danvar). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

File: src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/generate.cmd [hunk 1]
```diff
@@ -0,0 +1,17 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+set autoRestVersion=0.14.0-Nightly20160125
```
Comment (hovsepm) [inline src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/generate.cmd]: you should use at least 0.16.0-Nightly20160501

File: src/ResourceManagement/Cdn/Microsoft.Azure.Management.Cdn/project.json [hunk 2]
```diff
@@ -34,6 +34,8 @@
         "System.Diagnostics.Tools": "4.0.1-beta-23516",
         "System.Net.Http": "4.0.1-beta-23516",
         "System.Runtime.Serialization.Primitives": "4.1.0-beta-23516",
+        "System.Text.Encoding": "4.0.11-beta-23516",
+        "System.Text.RegularExpressions": "4.0.11-beta-23516",
```
Comment (hovsepm) [inline src/ResourceManagement/Cdn/Microsoft.Azure.Management.Cdn/project.json]: please revert these changes since they are not in your working folder.

Comment (daniel-ms) [inline src/ResourceManagement/Cdn/Microsoft.Azure.Management.Cdn/project.json]: reverted, sorry

File: src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/generate.cmd (hunk 1 above)
Comment (daniel-ms) [inline src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/generate.cmd]: changed.

**Comment by hovsepm** [comment]
@daniel-ms please remove src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/project.json file from your PR. And sync with Azure repo/AutoRest branch

**Comment by hovsepm** [comment]
:shipit:
